src/devices/Mcp25xxx/Register/AcceptanceFilter/RxFxSidh.cs

```diff
@@ -16,19 +16,19 @@ public class RxFxSidh : IRegister
         /// </summary>
         /// <param name="rxFilterNumber">
         /// Receive Filter Number.  Must be a value of 0 - 5.</param>
-        /// <param name="sid">
-        /// Standard Identifier Filter bits
+        /// <param name="standardIdentifierFilter">
+        /// SID[10:3]: Standard Identifier Filter bits
```

Comment:
not sure how to interpret those numbers in the brackets - initially thought: offset 10, count 3 but previously you've used 7:0

Reply:
Since renaming to extended name, I added the short name (along with respective bits) that is described with each property in datasheet.  It is not bits within the actual byte of register, but bits that make up the parameter.

Example...
In RxFxSidh, SID[10:3] represent bits 10-3 of the Standard Identifier and are 8 bits in this register.  In RxFxSidl, SID[2:0] represent bits 2-0 of the Standard Identifier and are the 3 higher bits in this register.  Standard Identifier is actually 11 bits long... so it is used across 2 registers.